manifests/3/3CX/3CXCallFlowDesigner/16.6.53/3CX.3CXCallFlowDesigner.installer.yaml

```
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: 3CX.3CXCallFlowDesigner
+PackageVersion: 16.6.53
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+  - cfdproj
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+Installers:
+  - Architecture: x64
+    InstallerType: exe
+    InstallerUrl: https://downloads-global.3cx.com/downloads/3CXCallFlowDesigner.exe
+    InstallerSha256: D852E35FAC3ACDFAEE2E127D8AE77724C1FCD952982296D4CDE933C82546E03D
+    InstallerSwitches:
+#      Silent:
```

```suggestion
      Silent: /qb
```
or /qn if that works